server/eclipse-project/script/rla_export/rla_export/default.properties

```diff
@@ -0,0 +1,27 @@
+# The default properties file for ColoradoRLA
+http_port = 8888
+https_port = 8889
+locale = en_US
+#keystore = /us/freeandfair/corla/default.jks
+#keystore_password = corla2017
+county_ids = us/freeandfair/corla/county_ids.properties
+# DatabaseAuthentiation or EntrustAuthentication
+authentication_class = us.freeandfair.corla.auth.DatabaseAuthentication
+# if authentication class is using EntrustAuthentication, then we must
+# specify the location of the Entrust server
+# entrust_server_name = localhost
+
+hibernate.driver = org.postgresql.Driver
+hibernate.url = jdbc:postgresql://localhost:5432/corla?reWriteBatchedInserts=true&disableColumnSantiser=true
+hibernate.user = corla
+hibernate.pass = corla
+hibernate.dialect = org.hibernate.dialect.PostgreSQL9Dialect
+hibernate.hbm2ddl.auto = update
+hibernate.show_sql = false
+hibernate.format_sql = false
+hibernate.use_sql_comments = false
+hibernate.c3p0.min_size = 5
```

Comment:
We need to be careful that attributes and values here reflect that which we determine is a reasonable config based upon current load testing.  We now have three or four different `default.property` files across the two repositories.

Reply:
Ahh - good to know that there's another one also - I was only aware of the one the server uses, and my intent was to fix the build process so that it picked it up from there.
I'm guessing that that will be ok, perhaps with a caveat in the file that it applies to a limited deployment scenario. I'm assuming they'll always want to specify a properties file. 

Now I'm thinking this whole `-p` option should be replaced by a config file option for choosing a local config file, which would point to the properties file and also include the password for the java server.